user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_parameters/I-DOCSIS_parameters_node_segment.md

```diff
@@ -70,10 +70,30 @@ This page contains an overview of the Node Segment parameters available in the I
 
 - **Percentage CM Reflection OOS**: Calculated. The percentage of cable modems associated with the given service group that are affected by reflection.
 
-- **Number CM Group Delay or Reflection OOS**: Calculated. The number of cable modems associated with the given service group that are affected by NMTER being out of spec (OOS).
+- **Number CM Group Delay or Reflection OOS**: Calculated. The number of cable modems associated with the given service group that are affected by Pre-MTTER being out of spec (OOS) or Post-MTTER being out of spec (OOS).
+
+- **Percentage CM Group Delay or Reflection OOS**: Calculated. The percentage of cable modems associated with the given service group that are affected by Pre-MTTER being out of spec (OOS) or Post-MTTER being out of spec (OOS).
+
+> [!NOTE]
+> Note that **Percentage CM Group Delay OOS** and **Percentage CM Reflection OOS** parameters were afected by NMTTER being out of spec, this was changed since **Percentage CM Group Delay or Reflection OOS** did not have relation between the two previous ones.
```

Comment:
What do you mean with "did not have relation between the two previous ones"? Could you rephrase this?